### PR TITLE
PostgreSQL database for inntekt API - PART 1

### PR DESCRIPTION
**Pull request by geiralund**
- Tables for inntekt DB
- Use flyway for database migration
- Use vault-jdbc to integrate against vault secrets for db
- kotlinquery as database library

### CODE REVIEW

File: src/test/kotlin/no/nav/dagpenger/inntekt/db/PostgresTest.kt [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    fun `JDBC url is set correctly from  config values `() {
+        with(hikariConfigFrom(Configuration())) {
+            assertEquals("jdbc:postgresql://localhost:5432/dp-inntekt-db", jdbcUrl)
+        }
+    }
+}
+
+internal class PostgresInntektStoreTest {
+    @Test
+    fun `Successful insert of inntekter`() {
```
Comment (gtcno) [inline src/test/kotlin/no/nav/dagpenger/inntekt/db/PostgresTest.kt]: Perhaps test that the mapping table has a a row as well?

File: src/main/kotlin/no/nav/dagpenger/inntekt/db/InntektStore.kt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            throw IllegalInntektId("ID $id is not a valid ULID", e)
+        }
+    }
+}
+
+class InntektNotFoundException(override val message: String) : RuntimeException(message)
+
+class StoreException(override val message: String) : RuntimeException(message)
+
+class IllegalInntektId(override val message: String, override val cause: Throwable?) : java.lang.RuntimeException(message, cause)
```
Comment (gtcno) [inline src/main/kotlin/no/nav/dagpenger/inntekt/db/InntektStore.kt]: Any reason for a fq name here(as opposed to the other exceptions?

Comment (geiralund) [reply src/main/kotlin/no/nav/dagpenger/inntekt/db/InntektStore.kt]: No, will add `Exception` postfix, a glitch

File: src/test/kotlin/no/nav/dagpenger/inntekt/db/PostgresTest.kt (hunk 1 above)
Comment (geiralund) [reply src/test/kotlin/no/nav/dagpenger/inntekt/db/PostgresTest.kt]: good point!

**Pull request merged**